documentation/staging/content/managing-domains/domain-lifecycle/retry.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Domain failure retry processing"
+date: 2022-10-10T08:14:51-05:00
+draft: false
+weight: 8
+description: "This document describes domain failure retry processing in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+{{< table_of_contents >}}
+
+The WebLogic Kubernetes Operator may encounter various failures during its processing of a Domain resource.
+Failures are are reported using Kubernetes events and the `spec.status` field in the Domain resource, 
+see [Domain debugging]({{< relref "/managing-domains/debugging#check-the-domain-status" >}}). 
+Failures fall into different categories and are handled differently by the operator, where most failures lead to automatic retries. 
+Refer to [Retry Behavior]({{< relref "#retry-behavior" >}}) on turning failure retry limits and intervals.
+
+### Domain failure severities
+
+Domain resource failures fall into the three severity levels:
+
+- Fatal failures
```

Comment:
Since fatal is rare, how about consider putting this in the order of how common each level occurs?

Warnings, Severe, Fatal instead of Fatal, Severe, Warnings.